scripts/format-code.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
This won't work on windows. Perhaps a python script would be better?
That said, if folk are working on windows they'll (probably) use GUI tools anyway over a script.